### PR TITLE
Update billing doc

### PR DESCRIPTION
**Pull request by DanRoscigno**
image

### CODE REVIEW

**Review by tbragin — APPROVED** [review]
LGTM

@DanRoscigno update: let's add a few examples

File: docs/en/manage/billing.md [hunk 1]
```diff
@@ -84,38 +88,37 @@ The only practical way to know exactly is to ingest your dataset into ClickHouse
 You can use the query `SELECT formatReadableSize(total_bytes) FROM system.tables WHERE name = <your table name>`. 
 
 ### What tools does ClickHouse offer to estimate the cost for running a service in the cloud if I have a self-managed deployment?
-The Query log captures key metrics in order to estimate the cost of running a workload in ClickHouse Cloud. Please contact ClickHouse Cloud support support@clickhouse.com with questions on migration.
+The ClickHouse query log captures [key metrics](/docs/en/operations/system-tables/query_log.md) that can be used to estimate the cost of running a workload in ClickHouse Cloud.  For details on migrating from self managed to ClickHouse Cloud please refer to the [migration documentation](/docs/en/integrations/migration/clickhouse-to-cloud.md), and contact ClickHouse Cloud support https://clickhouse.cloud/support if you have further questions.
 
 ### Do Backups count towards total storage?
```
Comment (tbragin) [inline docs/en/manage/billing.md:93]: "Backups" should be lower case

Comment (DanRoscigno) [inline docs/en/manage/billing.md:93]:
```suggestion
### Do backups count towards total storage?
```

**Comment by DanRoscigno** [comment]
Updated screenshot:
image

**Comment by tbragin** [comment]
@DanRoscigno Thank you for the update, LGTM

**Pull request merged**